src/main/java/redis/clients/jedis/BroadcastResponse.java

```diff
@@ -0,0 +1,25 @@
+package redis.clients.jedis;
+
+import java.util.function.Supplier;
+
+public class BroadcastResponse<T> implements Supplier<T> {
```

Comment:
I know we don't have a lot of them, but docstrings here would be a great start. Let's help users understand why these exist.